src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -785,6 +785,21 @@ static std::string parseNotifyConditionVector(ConnectionInfo* ciP, ngsiv2::Subsc
         subsP->subject.condition.expression.georel = georelOpt.value;
       }
     }
+
+    // If geometry, coords and georel are filled, then attemp to create a filter scope
```

Comment:
Fixed in 6a25b851c2eee240a24581294abd6ce500adfcd1

Not only in this file, but also in a couple of places that "replace all" has found.